desktop/main_core.js

```diff
@@ -283,6 +283,17 @@ function createDefaultMenu () {
     }
   });
 
+  menuHash['File'].push({
+    label: 'Open Backup Directory',
+    click() {
+        switch (process.platform) {
```

Comment:
No reason for this to be windows only.

Reply:
Oh, duh. This was left over from a thought process that the location would be OS specific, but clearly it's already dynamic based on where the project is located.